### ISSUE TITLE
Problem with popup windows

### ISSUE THREAD

**Issue by mefjak** [carenalgas/popochiu#49]
After adding a plugin to a project there is a problem with first setup windows.
Can't choose test resolution and can't close with close button.

Ver : Alpha-2.0.0, macOS, M1
<img width="327" alt="Zrzut ekranu 2023-04-17 o 15 17 16" src="https://user-images.githubusercontent.com/22888465/232496079-cb0aa651-cef3-437f-b5f5-69b9dc6522ca.png">

EDIT:
Problem when setting "Display Scale" to 200%
<img width="697" alt="Zrzut ekranu 2023-04-17 o 16 01 17" src="https://user-images.githubusercontent.com/22888465/232507282-632ecdf9-d1e6-4d7b-af4e-20f4546c2817.png">

**Comment by mefjak** [comment]
It's not only with first setup window
<img width="344" alt="Zrzut ekranu 2023-04-17 o 15 55 55" src="https://user-images.githubusercontent.com/22888465/232505849-3a28ae1b-0798-4ada-aeda-a54afc7f733e.png">

**Comment by stickgrinder** [comment]
@mapedorr I tested latest implementation on a 4K native-res Windows laptop.

The creation popups all seem to work correctly, but the setup has a problem: if you open it multiple times, it grows bigger and bigger. I tracked down the problem to `editor/popups/setup/setup.gd:73-74`, where you set min and max size by multiplying the values for the adjustment scale of the editor every time you show the window.

Maybe moving those two lines to `_ready()` will do the trick.
